src/Lib/Connector/Slim.php

```diff
@@ -64,12 +66,11 @@ protected function doRequest($request)
               ->withParsedBody($parsed);
         }
 
-        $slimHeaders = new Headers(['Content-Type' => 'text/html; charset=UTF-8']);
-
         $slimResponse = $this->app->process(
             $slimRequest,
-            (new Response(200, $slimHeaders))
-                ->withProtocolVersion($this->app->getContainer()->get('settings')['httpVersion'])
+            $container->get('response')
+              ->withProtocolVersion($container->get('settings')['httpVersion'])
```

Comment:
Looks like we could ommit this, because it was already defined when Slim create it's own default response [here](https://github.com/slimphp/Slim/blob/3.x/Slim/DefaultServicesProvider.php#L76).